### ISSUE TITLE
HELP!!! Close button doesn't work.

### ISSUE THREAD

**Issue by KKKyrie** [kamranahmedse/driver.js#263]
I use Driver.js in my Vue app, here's the code

```javascript
const driver = new Driver({
  closeBtnText: 'I knew',
  opacity: 0.5,
});

// some unrelated code

driver.highlight({
  element: '#button',
  popover: {
    title: '1',
    description: '123',
  },
});
```

My code is exactly the same as the official demo code, but I found that **the close button in popover** cannot close the overlay, even when I click the overlay, cannot close either.

I've checked whether I had execute `event.propagation()` method in my page, it turns out I haven't, every click inside my page can trigger the click event which I add to `document.body` element.

I was so confused. 😭 

Looking forward to your reply 🌹

**Comment by shengzhou1216** [comment]
save issue. but keyboard left/right arrow key can works.

**Comment by kamranahmedse** [comment]
We just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. This issue should no longer exist. Please have a [look at the docs](https://driverjs.com/) and do let me know if you face any issues.
